coralproject/talk
# Unified docs

## What does this PR do?

This PR gathers all of the Talk documentation into a navigable site based on Jekyll that will automatically be built and served by github pages.

## How do I test this PR?

```
cd docs
docker build --no-cache -t mydocs .
docker run -v "$PWD:/src" -p 4000:4000 mydocs serve -H 0.0.0.0
```
Browse to `localhost:4000` to see what they look like.
